### ISSUE TITLE
Live reactions for lectures

### ISSUE THREAD

**Issue by alexanderstephan** [joschahenningsen/TUM-Live#127]
**Is your feature request related to a problem? Please describe.**
When watching a lecture that uses Tweedback there are things like the panic button, that clearly indicate that the lecture is hard to follow. This can be really useful, since the chat can become hard to track for large amount of students.
That also makes it difficult to actually estimate what percentage of the student shares a certain opinion about the lecture. Therefore there should be an option for easy, instant, grouped feedback, that can also be visualized while being completely anonymous.

**Describe the solution you'd like**
There should be buttons that represent the mood of the student. For example there could be buttons that indicate the lecture moved too fast eg. 🏎️ , too slow e.g 🐌, or even mood indications. There could also be a certain threshold of votes e.g 20% of the current viewers within 5 minutes, that trigger an alert, that will be also directly sent to the lecturer.
The buttons should be combined with a heatmap or gauge, that will fade again after some time, as we expect the relevancy of the feedback to decrease over time. 

**Describe alternatives you've considered**
Of course you could always use Tweedback additionally, but it would be awesome if TUM-live became independent of other platforms and offers an integrated solution for lectures. Quizes can also be used for getting opinions about the lecture, but it is not ideal, as those have to be created manually.

**Comment by joschahenningsen** [comment]
Love this! If we want to take it up a notch there could also be an overview page for the lecturers that displays information on the lecture including:
- livestream preview (muted?)
- reactions and alerts
- live time left
- viewer count graph
- chat messages (would do this later because the chat will change a lot)
- ...

More advanced information could be:
- health status of the stream (useful for lecturers who use OBS)
- audio/video loss alerts (we can get this data from our hardware in the lecture halls)

But most of this is something for another PR I guess.
